apicast/src/oauth/keycloak.lua

```diff
@@ -237,9 +237,7 @@ function _M:authorize(service, client)
   end
 
   local url = resty_url.join(self.config.authorize_url, ngx.var.is_args, ngx.var.args)
-  local res = http_client.get(url)
-
-  _M.respond_and_exit(res.status, res.body, res.headers)
+  ngx.redirect(url)
```

Comment:
@mikz APIcast will not proxy the authorize endpoint, great. But what about the token endpoint ? This is my biggest concern currently about our OIDC implementation. Proxying the token endpoint, as we do currently, has limitations about flow customization or supportability. 

Any insight on the direction we are aiming ?